CONTRIBUTING.md

```diff
@@ -7,15 +7,31 @@ To that end, we follow the [18F Code of
 Conduct](https://github.com/18F/code-of-conduct/blob/master/code-of-conduct.md)
 and ask that all contributors do the same.
 
+### Not sure what to work on?
+
+Check out the [Dolores Bot Waffle.io
+board](https://waffle.io/18F/dolores-landingham-bot) and pick out something in
+the "Up next" column. Make sure you move the feature you're working on the "In
+progress" so we all know who is working on what.
+
 ### Git Protocol
 
-To contribute to this project, people internal to 18F can create a branch and submit a pull request. If you are external to 18F, you can fork the repository and submit a pull request that way.
+To contribute to this project, people internal to 18F can create a branch and
+submit a pull request. If you are external to 18F, you can fork the repository
+and submit a pull request that way.
 
-We are minimizing commits on the `master` branch by squashing commits and force pushing them to `master`. 
+We are minimizing commits on the `master` branch by rebasing and squashing
```

Comment:
Should we talk about doing a merge --squash of any given branch into master instead?  It's a simpler workflow and takes care of squashing all of your changes from the branch and staging them for a single new commit (but doesn't automatically commit them, you still do that yourself) into whatever branch you're merging into, e.g., master.